### PR TITLE
Merge history tables and refactor HistoryService

### PR DESCRIPTION
**Pull request by sprucely**
Also tweaked some controller actions to be a little more consistent WRT admin overrides.

### CODE REVIEW

File: Hippo.Web/Controllers/AccountController.cs [hunk 1]
```diff
@@ -221,12 +233,12 @@ public async Task<ActionResult> Create([FromBody] AccountCreateModel model)
 
         if (existingAccount != null && existingAccount.Status == Account.Statuses.Active)
         {
-            existingAccount.AccountYaml = await _yamlService.Get(currentUser, model, cluster);
-
-            await _historyService.AccountApproved(existingAccount);
-            await _historyService.AddHistory("Existing account override approve", $"Kerb: {existingAccount.Owner.Kerberos} IAM: {existingAccount.Owner.Iam} Email: {existingAccount.Owner.Email} Name: {existingAccount.Owner.Name}", existingAccount.ClusterId, existingAccount.Id);
+            if (existingAccount.Status != Account.Statuses.Active)
```
Comment (jSylvestre) [inline Hippo.Web/Controllers/AccountController.cs:236]: This can't be hit now?
Do we need/want to allow them to update their SSH before it is approved?

**Review by jSylvestre — APPROVED** [review]
Didn't test, but except for my one comment, looks good to me.

File: Hippo.Web/Controllers/AccountController.cs (hunk 1 above)
Comment (sprucely) [reply Hippo.Web/Controllers/AccountController.cs:236]: Oops. Forgot to change that above condition.

I believe the intent is that, if an account has already been approved, a user should be able to update their ssh without requiring further approval.

**Review by laholstege — APPROVED** [review]
didnt see anything beyond what jason pointed out 👍

**Pull request merged**